index.ts

```diff
@@ -45,11 +45,14 @@ class Client {
 
     const req = superagent.post(url.format(target)).send(data.body)
 
+    const bodyLength = JSON.stringify(data.body).length
```

Comment:
```suggestion
    const bodyLength = Buffer.byteLength(JSON.stringify(data.body))
```

see https://github.com/probot/smee-client/pull/152#issuecomment-894154072

We should still add a test before merging this change